style.css

```diff
@@ -12,6 +12,20 @@
   width: 100%;
 }
 
+.nav-menu {
+  margin-top: 0;
+  flex-direction: column;
+  width: 100%;
+  transition: 0.3s;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  color: #344563;
+}
+
```

Comment:
- [ ] You did a great job with the functionality of this mobile menu but in the mobile view, you have to side scroll to get the hamburger menu button so please fix the width of this menu to fix this problem.

![chrome-capture-2022-3-12](https://user-images.githubusercontent.com/68796232/162985412-d95474ce-3c74-4a9a-9160-93187e94bc4c.gif)